content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -227,7 +227,10 @@ These default values of hard eviction thresholds will only be set if none
 of the parameters is changed. If you change the value of any parameter,
 then the values of other parameters will not be inherited as the default
 values and will be set to zero. In order to provide custom values, you
-should provide all the thresholds respectively.
+should provide all the thresholds respectively Or you can set the Kubelet Config
+MergeDefaultEvictionSettings to true in the Kubelet Configuration file.
```

Comment:
```suggestion
MergeDefaultEvictionSettings to true in the kubelet configuration file.
```

Reply:
@natalisucks I have updated the suggested change